eeg_matchingpennies/task-matchingpennies_events.json

```diff
@@ -29,38 +25,62 @@
+        "Units": "s"
     },
-    "latency": {
-        "Description": "Estimated beginning of the data chunk used for the BCI prediction with respect to the true event as measured by parallel port trigger in this trial.",
+    "countdown_offset": {
+        "Description": "Offset of the countdown. Must raise a hand now.",
+        "Units": "s"
+    },
+    "response_time": {
+        "Description": "Reaction time to lift the left or right hand after the countdown reached zero.",
         "Units": "ms"
     },
-    "trial": {
-        "Description": "The current trial within this stage."
+    "feedback_onset_approx": {
+        "Description": "Approximate onset of the bci-produced feedback (either a left or a right hand visual stimulus).",
+        "Units": "s"
+    },
+    "stim_file": {
+        "Description": "Name of the stimulus file that was shown during this trial at 'feedback_onset_approx'."
+    },
+    "trial_type": {
+        "Description": "Which hand was raised, and did this match the BCIs prediction (lose outcome), or not (win outcome).",
+        "Levels": {
+            "raised-left/match-false": "raised left hand, bci predicted right hand",
+            "raised-left/match-true": "raised left hand, bci predicted left hand",
+            "raised-right/match-false": "raised right hand, bci predicted left hand",
+            "raised-right/match-true": "raised right hand, bci predicted right hand"
+        },
+        "HED": {
+            "raised-left/match-true": "Description/Subject raised left hand and computer presented image of left hand, (Agent-action, Experiment-subject, (Lift, (Left-side, Hand)), (Release, (Left-side, Button)), ((Delay/68 ms, Approximated), (Sensory-presentation, Visual), (Drawing, ID/left_hand.png, (Left-side, Hand)), (Feedback, (Intended-effect, Penalty))))",
+            "raised-left/match-false": "Description/Subject raised left hand and computer presented image of right hand, (Agent-action, Experiment-subject, (Lift, (Left-side, Hand)), (Release, (Left-side, Button)), ((Delay/68 ms, Approximated), (Sensory-presentation, Visual), (Drawing, ID/right_hand.png, (Right-side, Hand)), (Feedback, (Intended-effect, Reward))))",
+            "raised-right/match-true": "Description/Subject raised right hand and computer presented image of right hand, (Agent-action, Experiment-subject, (Lift, (Right-side, Hand)), (Release, (Right-side, Button)), ((Delay/68 ms, Approximated), (Sensory-presentation, Visual), (Drawing, ID/right_hand.png, (Right-side, Hand)), (Feedback, (Intended-effect, Penalty))))",
+            "raised-right/match-false": "Description/Subject raised right hand and computer presented image of left hand, (Agent-action, Experiment-subject, (Lift, (Right-side, Hand)), (Release, (Right-side, Button)), ((Delay/68 ms, Approximated), (Sensory-presentation, Visual), (Drawing, ID/left_hand.png, (Left-side, Hand)), (Feedback, (Intended-effect, Reward))))"
+        }
+    },
```

Comment:
@VisLab can you help me finding the correct HED vocabulary for what I want to express here? These tags are based on a draft you sent me several months ago - and now that HED 8.0.0 is released (:rocket:), the HED validator is telling me that some of these tags are not valid.

See also the report here: https://github.com/bids-standard/bids-examples/pull/265/checks?check_run_id=3280815398#step:8:1584

and copy paste of the first lines:

```
		./task-matchingpennies_events.json
			Evidence: ERROR: [HED_TAG_INVALID] Invalid tag - "Experiment-subject"
		./task-matchingpennies_events.json
			Evidence: ERROR: [HED_TAG_INVALID] Invalid tag - "Left-side"
		./task-matchingpennies_events.json
			Evidence: ERROR: [HED_TAG_INVALID] Invalid tag - "Left-side"
		./task-matchingpennies_events.json
			Evidence: ERROR: [HED_TAG_INVALID] Invalid tag - "Button"
		./task-matchingpennies_events.json
			Evidence: ERROR: [HED_TAG_INVALID] Invalid tag - "Approximated"
		./task-matchingpennies_events.json
			Evidence: ERROR: [HED_TAG_INVALID] Invalid tag - "Visual"
		./task-matchingpennies_events.json
			Evidence: ERROR: [HED_TAG_INVALID] Invalid tag - "Left-side"
		./task-matchingpennies_events.json
			Evidence: ERROR: [HED_TAG_INVALID] Invalid tag - "Delay/68 ms, Appro"
		./task-matchingpennies_events.json
			Evidence: ERROR: [HED_TAG_INVALID] Invalid tag - "(Sensory-"
		./task-matchingpennies_events.json
			Evidence: ERROR: [HED_TAG_INVALID] Invalid tag - ", (Drawin"
```

Reply:
I took some time this morning and I think it's fine now, see https://github.com/bids-standard/bids-examples/pull/265#issuecomment-1013855511